packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -123,4 +124,8 @@ export class AtomicResult {
       ></div>
     );
   }
+
+  public componentDidLoad() {
+    applyFocusVisiblePolyfill(this.host);
```

Comment:
Do you think there are other type of components that would require this "special" treatment of also calling `applyFocusVisiblePolyfill` instead of the generic one applied to all components in initialization-utils ?

I guess not...

Reply:
Checked and no 😃